UnitsNet.Tests/UnitSystemTests.cs

```diff
@@ -257,6 +257,64 @@ public string ScientificNotationUpperIntervalFormatting(double value)
         }
 
         [Test]
+        public void ShouldUseCorrectMicroSign()
+        {
```

Comment:
Nice set of tests this, even though it will get out of date at some point it is a nice sanity check.